tools/pipelines/build-docs.yml

```diff
@@ -174,12 +174,20 @@ stages:
           inputs:
             command: 'custom'
             workingDir: $(Build.SourcesDirectory)/docs
-            customCommand: 'run ci:build'
+            customCommand: 'run build'
+
+        # Run the tests
```

Comment:
Just realized we have the ADO pipeline and the github workflow. Is one of them redundant?